CHANGELOG

```diff
@@ -67,6 +67,8 @@
 
 - add a new ``--noconftest`` argument which ignores all ``conftest.py`` files.
 
+- add ``file`` and ``line`` attributes to JUnit-XML output.
```

Comment:
If you want, feel free to add a *Thanks to Kevin Cox for PR" like some of the other entries. :wink: